Adds missing information of named locations

bltavares opened the pull request:
This pull request adds information about name locations where it is possible to capture Pokemons.

Previously, the checklist had all the information about the numbered routes, but to complete the whole Alola pokedex, we need to check some other locations beyond the numbered routes.

So I've used Bulbapedia to visit the missing "named locations" on each island, and add the information about the Pokemons. With this, the checklist should be have all habitats and even more helpful for those that want to complete the regional dex.

[Final result Demo here](https://blog.bltavares.com/pokefinder/)

Closes issue #13

bltavares committed:
Adds island information

This commit adds the information of which island this route is related to, to help check the captured pokemons on the Pokedex

bltavares committed:
Adds Ten Carat Hill pokemon data

bltavares committed:
Adds more information about melemele island

bltavares committed:
Fix typo on location's name

bltavares committed:
Adds information about Hau'oli Cemetery

bltavares committed:
Adds remaining information about first island

bltavares committed:
Port the information about the first island to the night mode

bltavares committed:
Adds information about island 2

bltavares committed:
Port information to night mode

bltavares committed:
Adds information about pokemons on island 3

bltavares committed:
Adds more information about the Ula'ula island

bltavares committed:
Separates the forms of Oricorio for each checkbox to be checked separatelly

bltavares committed:
Port the information to the night version

bltavares committed:
Adds all the information about the Poni island encounters

bltavares committed:
Port the information into the night theme

GiacomoLaw commented:
Is it possible to sort the missing locations under island? Also the routes?

Thanks for your PRs!

GiacomoLaw commented:
The header is also getting a little cluttered, so it would be handy if it was just islands and routes on there.

bltavares commented:
The locations and routes are asked by the order you visit then during the story, so if you are at Route 1 you can go ahead and visit the other locations in order to fill the Pokedex.

I'll see what I can do about the headers. 

On September 26, 2017 4:44:56 AM GMT-03:00, Giacomo Lawrance <notifications@github.com> wrote:
>Is it possible to sort the missing locations under island? Also the
>routes?
>
>Thanks for your PRs!
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/GiacomoLaw/pokefinder/pull/16#issuecomment-332114447

GiacomoLaw commented:
Awesome, thanks

bltavares committed:
Organize the menu and improve the whitespace usage

bltavares commented:
@GiacomoLaw I've updated the style of the menu to make it more regular. I've also fiddled a bit with the whitespace on the checklist boxes so we can better use the space around when it is on a big screen.

This changes are considering mobile browsers, so it is a big list when using on a small screen.
I've updated my [demo](https://blog.bltavares.com/pokefinder/) page if you want to try on mobile easily. (It should take a while until the CSS cache finishes)

Let me know what you think of it.

bltavares committed:
Adjust font-size on small screens to make the menu readable

bltavares committed:
Removes manual cache-bust

GiacomoLaw reviewed (changes requested):
Love the layout. Dropdowns for the headers would be awesome, and the header text is tiny on desktop.

The top of the page with the header linking to all the sections is really cramped. A dropdown box with the routes and areas divided by islands would be awesome.

Thanks for all your hard work, really appreciate it!

bltavares commented:
Would you mind sharing a screenshot and the resolution you used so I can try it out? It was looking ok on my laptop and cellphone.

Changing to drop-down on the top menu would require more time and effort. What do you think of getting the new information merged, and then adjust it?
I would be worried to have it stale and maybe someone with better css skills would be able to help with the drop-down.


On September 26, 2017 12:44:26 PM GMT-03:00, Giacomo Lawrance <notifications@github.com> wrote:
>GiacomoLaw requested changes on this pull request.
>
>Love the layout. Dropdowns for the headers would be awesome, and the
>header text is tiny on desktop.
>
>The top of the page with the header linking to all the sections is
>really cramped. A dropdown box with the routes and areas divided by
>islands would be awesome.
>
>Thanks for all your hard work, really appreciate it!
>
>
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/GiacomoLaw/pokefinder/pull/16#pullrequestreview-65263394

GiacomoLaw commented:
![screenshot 2017-09-26 17 19 41](https://user-images.githubusercontent.com/9038673/30871473-ecf60e08-a2de-11e7-8e6e-79764394762d.png)

That's a screenshot from my PC

bltavares committed:
Increase the font-size of the menu

bltavares commented:
I've increased the font size on the menu and [here is how it is looking](https://imgur.com/a/5ES8N)

I might work on the dropdown another time tho.

bltavares committed:
Converts the top navigation menu into grouped dropdown lists of locations for each island

bltavares commented:
@GiacomoLaw I've set up a dropdown menu, which might not be the prettiest but I believe it delivers a better organization. I've tested it on my cellphone as well and it seems to be good for my usecase.

The demo might take a while to refresh the cache. Let me know what you think of it.

GiacomoLaw commented:
Just looked at it, the dropdown is really good. Thanks so much for your commits, merged!

bltavares commented:
Thank you for merging and I hope you liked.

Now I'll be back into capturing Pokemons

Cheers!

On September 27, 2017 5:35:56 AM GMT-03:00, Giacomo Lawrance <notifications@github.com> wrote:
>Merged #16.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/GiacomoLaw/pokefinder/pull/16#event-1267294005
